pkg/subprocess/subprocess_test.go

```
@@ -51,7 +51,7 @@ func init() {
 func helperCommand(command string, args ...string) *exec.Cmd {
 	allArgs := []string{"-test.run=^$", "--", command}
 	allArgs = append(allArgs, args...)
-	cmd := exec.Command(os.Args[0], allArgs...)
+	cmd := exec.Command(os.Args[0], allArgs...) //nolint:gosec // test helper re-invoking own binary
```

golangci-lint (2.12.2)

[error] os/exec.Command must not be called. use os/exec.CommandContext

(noctx)